ibm_catalog.json

```diff
@@ -8,7 +8,7 @@
                     "label": "SAP prepared PowerVS instances",
                     "name": "prepared_system",
                     "working_directory": "examples/ibm-catalog/prepared-system-solution",
-                    "usage": "module \"sap_systems\" {\n  source                    = \"https://cm.globalcatalog.cloud.ibm.com/api/v1-beta/offering/source//examples/ibm-catalog/prepared-system-solution?archive=tgz&kind=terraform&name=terraform-ibm-powervs-catalog-powervs-sap-infrastructure&version=1.0.5\"\n  ibmcloud_api_key          = var.ibmcloud_api_key\n  prerequisite_workspace_id = var.prerequisite_workspace_id\n  powervs_zone              = var.powervs_zone\n  prefix                    = var.prefix\n  ssh_private_key           = var.ssh_private_key\n}",
+                    "usage": "TBD",
```

Comment:
Why is this TBD? Ideally this needs to be filled in before the publish to catalog

Reply:
reverted it back. Had a conflict from other branch. Thank you